src/main/java/duke/parser/TaskParser.java

```diff
@@ -0,0 +1,20 @@
+package duke.parser;
+
+import duke.task.Task;
+
+/**
+ * This is a parser used to parse individual tasks into a string representation that will be saved into the file path
+ * specified in {@link duke.main.Duke} and {@link duke.storage.Storage}.
+ */
+public class TaskParser {
+
+    /**
```

Comment:
As this class only contains one method which is parse(), it could perhaps be done away with. You could also consider shifting the parse method into the Task class, where it can be used like `Task.parse()` without having to specify the task in the parameters. :)